system/settings/settings.xml

```diff
@@ -1434,6 +1434,17 @@
             <formatlabel>14044</formatlabel>
           </control>
         </setting>
+        <setting id="pvrrecord.preventduplicateepisodes" type="integer" label="812" help="36423">
```

Comment:
do we need a global setting for this? I suppose it's to define the default for new repeating timers? Maybe adjust the label/description accordingly (like we also have a "default start padding time" and "start padding time" added by this PR). Can ofc also be fixed later, but should not be forgotten. If label is adjusted, then the timer dialog itself needs a separate label ofc (that doesn't contain anything "default")